src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -257,7 +257,14 @@ private static void AddBracketIndentationOperation(List<IndentBlockOperation> li
             if (!bracketPair.IsValidBracketOrBracePair())
                 return;
 
-            if (node.Parent != null && node.Kind() is SyntaxKind.ListPattern or SyntaxKind.CollectionExpression)
+            if (node.Parent != null && node.Kind() is SyntaxKind.ListPattern)
```

Comment:
@CyrusNajmabadi I think the safest way is to separate `ListPattern` from `CollectionExpression` and use the logic before [this change](https://github.com/dotnet/roslyn/pull/70060/files#diff-00ba6c989b4980472de12cb6baa8bc95adf9d09b93dfbe7fe2e4c23d6e348592).

As I know list pattern doesn't support wrapping and we don't need to handle the double indentation issue.